project/Dependencies.scala

```diff
@@ -4,10 +4,10 @@ object Dependencies {
 
   val test = Seq(
     "uk.gov.hmrc"                %% "webdriver-factory"       % "0.18.0" % Test,
```

Comment:
Can we go up to the following versions of webdriver-factory and zap-automation? 
![Screenshot 2021-06-01 at 11 56 47](https://user-images.githubusercontent.com/28566263/120312620-e1126f80-c2d0-11eb-886c-e4f3d7bc2c12.png)

Reply:
Updated - not sure how i missed those